app/uk/gov/hmrc/economiccrimelevyregistration/models/EntityType.scala

```diff
@@ -20,15 +20,15 @@ import play.api.libs.json._
 
 sealed trait EntityType
 
-case object UkLimitedCompany extends EntityType
-case object SoleTrader extends EntityType
-case object GeneralPartnership extends EntityType
-case object ScottishPartnership extends EntityType
-case object LimitedPartnership extends EntityType
-case object ScottishLimitedPartnership extends EntityType
-case object LimitedLiabilityPartnership extends EntityType
-
 object EntityType {
+  case object UkLimitedCompany extends EntityType
+  case object SoleTrader extends EntityType
+  case object GeneralPartnership extends EntityType
+  case object ScottishPartnership extends EntityType
+  case object LimitedPartnership extends EntityType
+  case object ScottishLimitedPartnership extends EntityType
+  case object LimitedLiabilityPartnership extends EntityType
+
```

Comment:
Missing 'Other' entity type (which I assume will be needed here as, if selected, will be saved against the registration?)